Sources/Common/DataModel/ImplicitBoolean/index.js

```diff
@@ -19,6 +19,18 @@ function vtkImplicitBoolean(publicAPI, model) {
   // Set our className
   model.classHierarchy.push('vtkImplicitBoolean');
 
+  publicAPI.getMTime = () => {
+    let mTime = model.superClass.getMTime();
```

Comment:
Did you bound the superClass on the model? 
It should be a closure/private variable not something shared across all the hierarchy via the model.